soccer/KickEvaluator.cpp

```diff
@@ -96,9 +96,8 @@ KickResults KickEvaluator::eval_pt_to_seg(Point origin, Segment target) {
         // If robot is past target, only use the chance at the target segment
         if (distPastTarget > 0 && fabs(get<1>(loc)) < M_PI / 2) {
             // Evaluate a normal distribution at dist away and scale
-            float stdev2 = pow(*robot_std_dev, 2);
             botVertScales.push_back(
-                fast_exp(-0.5 * pow(distPastTarget, 2) / stdev2));
+                1 - erf(distPastTarget / (*robot_std_dev * sqrt(2)));
```

Comment:
I believe you're missing a `(1 +` here

Reply:
If you try and run `make run sim` it doesn't compile because of the parantheses

Reply:
So I think this is actually correct. In the issue, I messed up the formula. It was corrected in #1064 before we closed it